app/src/test/java/nerd/tuxmobil/fahrplan/congress/utils/ConferenceTimeFrameTest.kt

```diff
@@ -48,59 +48,59 @@ class ConferenceTimeFrameTest {
     }
 
     @Test
-    fun firstDayStart() {
+    fun `firstDayStart returns the the first day`() {
```

Comment:
returns the (-the) first day start time

Reply:
:heavy_check_mark: Thank you. Fixed in d13848f51f4f4d92e8fd463fbccee6ced1808fb3.